common.mk

```diff
@@ -2,19 +2,20 @@ export PYLINT = flake8
 export UTILS_DIR = "$(INDRA_HOME)/utils"
 PYTHONFILES = $(shell ls *.py)
 PYLINTFLAGS = --max-returns-amount=4 --max-parameters-amount=12 --max-function-length=40
+NOSEFLAGS = --exe --verbose
 export user_type = test
 
 FORCE:
 
 tests: pytests lint
 
 pytests: FORCE
-	nosetests --exe --verbose --with-coverage --cover-package=$(PKG)
+	nosetests $(NOSEFLAGS) --with-coverage --cover-package=$(PKG)
```

Comment:
running`make tests` results in $PKG being empty. 
It lists coverage for pip modules as well.
Maybe we can default this to Indra packages or remove it if $PKG is blank.